audio/engine.asm

```diff
@@ -353,7 +353,7 @@ UpdateChannels:
 	ldh a, [rNR52]
 	and %10001101 ; ch2 off
 	ldh [rNR52], a
-	ld hl, rNR20
+	ld hl, $ff15 ; rNR20 (imaginary)
```

Comment:
There's nothing imaginary about the sound registers: https://gbdev.io/pandocs/Sound_Controller.html

If hardware.inc isn't going to add them, they can go in audio_constants.asm.

Reply:
They are nonexistant in the hardware itself, but if you insist

Reply:
They are nonexistant in the hardware itself, but if you insist

Reply:
`NR20` is [very much imaginary](https://gbdev.io/pandocs/Sound_Controller.html#sound-channel-2---tone).